docs/src/intro.md

```diff
@@ -1,44 +1,17 @@
 # Introduction
 
-_Approximate Lower Bound Arguments_ are a form of cryptographic certificates that allow a _prover_ to convince a _verifier_ they know some large set of elements by providing only a small subset of those elements. This excellent [X thread](https://x.com/Quantumplation/status/1783188333046255997) provides a good intuition on why ALBAs are useful and how they work, and details of the theory behind this construction are beyond the scope of this introduction and can be found in the paper.
+## Definition
 
-### Proving
+_Approximate Lower Bound Arguments_ are a form of cryptographic certificates that allow a _prover_ to convince a _verifier_ they know some set of elements by providing only a small subset of those elements. More formally, given some set of _unique_ elements, and some bounds \\(n_p\\) and \\(n_f\\), where \\(n_p > n_f\\), about the size of this set, ALBA provides an algorithm that allows to build a _proof_ they know _at least_ \\(n_f\\) elements, where the proof size is a constant value derived from the \\(\frac{n_p}{n_f}\\) ratio.
```

Comment:
> given some set of _unique_ elements

What do you mean by this? What would be an example of a set of *non-unique* elements?

Reply:
Silly me 🤦

Reply:
this is my dumb rephrasing of the ALBA requirement that $S_p$ must be indeed a set, so that one cannot grow arbitrarily the size of items to construct a proof from. This would be a problem in the case of simple Ed25519 signatures because, as I understand from discussions with cryptographers, anyone can create more valid signatures from a single signature. this is not a problem in general, but for ALBA it obviously is.

Reply:
Ah, makes sense, so this paragraph from section 1.3 from the paper:

![2024-10-01_22-41](https://github.com/user-attachments/assets/9b8cb12e-251d-4ba9-ac4b-fa73b9190406)

That indeed seems worth to point out somewhere in more detail :+1: